libsolidity/parsing/Parser.cpp

```diff
@@ -1831,7 +1831,25 @@ ASTPointer<TypeDefinition> Parser::parseTypeDefinition()
 	if (m_scanner->currentToken() == Token::Assign)
 	{
 		expectToken(Token::Assign);
-		expression = parseExpression();
+
+		if (m_scanner->currentToken() != Token::Builtin)
+			expression = parseExpression();
+		else
+		{
+			expectToken(Token::Builtin);
+			expectToken(Token::LParen);
+
+			// TMP: Do I need to guard this?
+			//if (m_scanner->currentToken() == Token::StringLiteral)
+			expression = nodeFactory.createNode<BuiltinDefinition>(
+				std::make_shared<std::string>(m_scanner->currentLiteral()),
+				// TMP: Verify that it's the right location
+				m_scanner->currentLocation()
+			);
+
+			expectToken(Token::StringLiteral);
+			expectToken(Token::RParen);
```

Comment:
Not sure if this is the best way to parse it:
- Maybe I should allow arbitrary arguments and only reject invalid ones in the analysis?
- Even if not, maybe the name of the builtin should be parsed into a `Literal` node? Currently I only expect a string and pass it as such to the `BuiltinDefinition` AST node.
    - If it's a literal, I can make AST visit it. Not sure if that would be useful though.